include/albatross/src/evaluation/prediction_metrics.hpp

```diff
@@ -121,6 +121,16 @@ struct NegativeLogLikelihood : public PredictionMetric<PredictType> {
   NegativeLogLikelihood()
       : PredictionMetric<PredictType>(negative_log_likelihood) {}
 };
+
+inline double chi_squared_cdf(const JointDistribution &prediction,
+                              const MarginalDistribution &truth) {
+  return chi_squared_cdf(prediction.mean - truth.mean, prediction.covariance);
```

Comment:
What are the merits/negatives to adding the `truth.diagonal` to the prediction.covariance to the second argument? If the truth is low certainty, any errors would be low certainty aswell.

Reply:
Yeah I went back and forth on this.  Though you're swaying me back again.  On one hand the prediction might contain measurement noise already, but yeah the truth may also have a different type of measurement noise.  It's probably worth capturing that here.  I'll change it.